src/login/oidc/oidcHandlers/LegacyImplicitFlowOidcHandler.ts

```diff
@@ -38,15 +40,22 @@ export default class LegacyImplicitFlowOidcHandler implements IOidcHandler {
     @inject("sessionInfoManager")
     private sessionInfoManager: ISessionInfoManager,
     @inject("redirector") private redirector: IRedirector,
-    @inject("dpopHeaderCreator") private dpopHeaderCreator: IDpopHeaderCreator
+    @inject("dpopClientKeyManager")
+    private dpopClientKeyManager: IDpopClientKeyManager,
+    @inject("dpopHeaderCreator") private dpopHeaderCreator: IDpopHeaderCreator,
+    @inject("storageUtility") private storageUtility: IStorageUtility
   ) {}
 
   async canHandle(oidcLoginOptions: IOidcOptions): Promise<boolean> {
     return !!(
       oidcLoginOptions.issuerConfiguration.grantTypesSupported &&
       oidcLoginOptions.issuerConfiguration.grantTypesSupported.indexOf(
         "implicit"
-      ) > -1
+      ) > -1 &&
+      // Escape hatch to detect that we are talking to NSS and not the id broker
```

Comment:
Yike! Yeah, big TODO: or FIXME: needed here (as with Jack's comment above on `secure: false` (which occurs twice)).

Reply:
Addressed in e65ae11bb5e81297c1fbc3e19e0219f4d3b5282d